### PR TITLE
Adding admin attack notice

### PR DESCRIPTION
**Pull request by rjtwins**
:cl: rjtwins
add: Admin attack notice.
/:cl:

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -73,7 +73,6 @@
 #include "code\__HELPERS\lists.dm"
 #include "code\__HELPERS\maths.dm"
 #include "code\__HELPERS\matrices.dm"
-#include "code\__HELPERS\mobs.dm"
```
Comment (halworsen) [inline tgstation.dme]: You shouldn't touch tgstation.dme. Leaving it alone prevents merge conflicts when we update from /tg/

File: modules_artemis/__HELPERS/mobs.dm [hunk 2]
```diff
@@ -0,0 +1,409 @@
+/proc/random_blood_type()
```
Comment (halworsen) [inline modules_artemis/__HELPERS/mobs.dm]: Lots of undocumented changes

File: modules_artemis/__HELPERS/mobs.dm [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+	for(var/i=1, i<=attempts_to_find_unique_name, i++)
+		if(gender==FEMALE)
+			. = capitalize(pick(first_names_female)) + " " + capitalize(pick(last_names))
+		else
+			. = capitalize(pick(first_names_male)) + " " + capitalize(pick(last_names))
+
+		if(i != attempts_to_find_unique_name && !findname(.))
+			break
+
+/proc/random_unique_lizard_name(gender, attempts_to_find_unique_name=10)
```
Comment (halworsen) [inline modules_artemis/__HELPERS/mobs.dm]: You should make these procs of the species datums.
E.g. /datum/species/lizard/random_name()

File: modules_artemis/__HELPERS/mobs.dm [hunk 4]
```diff
@@ -0,0 +1,409 @@
...
+	for(var/i=1, i<=attempts_to_find_unique_name, i++)
+		. = capitalize(plasmaman_name())
+
+		if(i != attempts_to_find_unique_name && !findname(.))
+			break
+
+/proc/random_skin_tone()
+	return pick(skin_tones)
+
+var/list/skin_tones = list(
```
Comment (halworsen) [inline modules_artemis/__HELPERS/mobs.dm]: defining a proc using a variable before you define the variable triggers me

(pls move the list definition to above the proc)

File: modules_artemis/__HELPERS/mobs.dm [hunk 5]
```diff
@@ -0,0 +1,409 @@
...
+		if(45 to 60)
+			return "middle-aged"
+		if(60 to 70)
+			return "aging"
+		if(70 to INFINITY)
+			return "elderly"
+		else
+			return "unknown"
+
+/*
```
Comment (halworsen) [inline modules_artemis/__HELPERS/mobs.dm]: why is this code grouped together with character randomization procs?
a new file dedicated to admin tool helpers would be a better choice IMO

**Review by halworsen — DISMISSED** [review]
There are undocumented changes, some code organization that triggers me and changes to the tgstation.dme.

**Comment by rjtwins** [comment]
Atebite Its a global proc I'm editing I've unticked the dm file from the tg load order in order to modify it.
There are no changes to the actual code apart from a line added.

**Comment by halworsen** [comment]
can't we use this as an excuse to make the code better then

**Comment by rjtwins** [comment]
Fair enough, I've modified it to not change any original code.

**Pull request merged**